app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -118,16 +174,15 @@ class RecentlyPlayedFragmentTest {
         )
       ).check(
         matches(
-          withText(containsString("The Meaning of Equal Parts"))
+          withText(containsString("What is a Fraction?"))
```

Comment:
Please use the reference of this text from string.xml resource file instead of hard-coding the text..

Reply:
Actually we do use `Hard-coded` strings in tests. This is how it works:
1. If there is a string which in `strings.xml` then we should use that string and do not use a hard-coded string.
2. If the string is not present in `strings.xml` then hardcode it but do not introduce that in `strings.xml` file as we do not want that string to be used in `app` module.

Reply:
Ok